### PR TITLE
Updates Week 5 : AI-Tools

### PR DESCRIPTION
**Pull request by Manas23601**
## Project

Deployment Optimization

### Changes

- Added weekly updates
- GPU support for docker containers in swarm

### Learnings 

- learnt more about nginx configuration parameters. Experimented with alternate configuration settings.
- learnt about docker networks and volumes.

### CODE REVIEW

**Review by Apoorvgarg-creator — CHANGES_REQUESTED** [review]
Don't leave blank headers

File: docs/2023/AI Tools/Deployment Optimization/updates/2023-08-04.md [hunk 1]
```diff
@@ -4,13 +4,9 @@ author: Manas Sivakumar
...
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
-
-## Screenshots / Videos 
-
+- Fix generate.sh and ansible script issues
+  
 ## Contributions
-
```
Comment (shubhigupta991) [inline docs/2023/AI Tools/Deployment Optimization/updates/2023-08-04.md:15]: Please add your contribution in terms of PR.

File: docs/2023/AI Tools/Deployment Optimization/updates/2023-08-04.md [hunk 2]
```diff
@@ -4,13 +4,9 @@ author: Manas Sivakumar
...
-## Learnings
\ No newline at end of file
+- Tried to setup nginx server
+  
+## Learnings
```
Comment (shubhigupta991) [inline docs/2023/AI Tools/Deployment Optimization/updates/2023-08-04.md:12]: Add learnings.

**Review by shubhigupta991 — CHANGES_REQUESTED** [review]

File: docs/2023/AI Tools/Deployment Optimization/updates/2023-08-04.md (hunk 1 above)
Comment (Manas23601) [reply docs/2023/AI Tools/Deployment Optimization/updates/2023-08-04.md:15]: I don't have any prs for this week

**Pull request merged**